File: javascript/createPhoneNumber.js
function createPhoneNumber(numbers){

    const x =  numbers.reduce((acc, cur) => acc.toString() + cur.toString());
    return `(${x.slice(0, 3)}) ${x.slice(3, 6)}-${x.slice(6, 10)}`;

}

console.log(createPhoneNumber([1, 2, 3, 4, 5, 6, 7, 8, 9, 0]));

File: python/6_kyu/create_phone_number.py
from functools import reduce
def create_phone_number(n):
    x = reduce(lambda x, y: str(x) + str(y), n)
    return f"({x[:3]}) {x[3:6]}-{x[6:]}"

# def create_phone_number(n):
#     x = ""
#     for k in n: x += str(k)
#
#     return f"({x[:3]}) {x[3:6]}-{x[6:]}"
    
print(create_phone_number([1, 2, 3, 4, 5, 6, 7, 8, 9, 0]))
